### ISSUE TITLE
{{tts fr_FR:L2}} crashes Windows anki.

### ISSUE THREAD

**Issue by fuquasteve** [ankitects/anki#1316]
L2 is a defined field.

Whenever

{{tts fr_FR:L2}}

Is on the front of the card 

whenever I open the card to edit it or when I open it to study it
produces:

Error 
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed. 
If the issue only occurs when add-ons are enabled, please use the Tools>Add-ons menu item to disable some add-ons and restart Anki, repeat until you discover the add-on that is causing the problem. 
When you've discovered the add-on that is causing the problem, please report the issue on the add-ons section of our support site. 
Debug info:
Anki 2.1.44 (b2b3275f) Python 3.8.6 Qt 5.14.2 PyQt 5.14.2
Platform: Windows 10
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2021-07-29 19:58:59

Caught exception:
Traceback (most recent call last):
  File "aqt\progress.py", line 53, in handler
  File "aqt\clayout.py", line 517, in _renderPreview
  File "aqt\sound.py", line 152, in play_tags
  File "aqt\sound.py", line 203, in _play_next_if_idle
  File "aqt\sound.py", line 206, in _play
  File "aqt\sound.py", line 217, in _best_player_for_tag
  File "aqt\tts.py", line 107, in rank_for_tag
  File "aqt\tts.py", line 72, in voice_for_tag
  File "aqt\tts.py", line 68, in voices
  File "aqt\tts.py", line 486, in get_available_voices
  File "aqt\tts.py", line 491, in _voice_to_object
  File "<COMObject <unknown>>", line 3, in GetAttribute
pywintypes.com_error: (-2147352567, 'Exception occurred.', (0, None, None, None, 0, -2147200966), None)

This happens even with all add-ons removed from the program.  This happens every single time.
In case there is more information from the "about" listings on the ### programs and on the computer...
I am running Version ⁨2.1.44 (b2b3275f)⁩
Python 3.8.6 Qt 5.14.2 PyQt 5.14.2

on 

Device name	DESKTOP-FJCMFED
Processor	Intel(R) Core(TM) i5-1035G1 CPU @ 1.00GHz   1.19 GHz
Installed RAM	8.00 GB (7.77 GB usable)
Device ID	9813CED2-0923-4679-ABEE-AE2FEDDF4637
Product ID	00325-81788-88104-AAOEM
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

using

Edition	Windows 10 Home
Version	20H2
Installed on	‎3/‎13/‎2021
OS build	19042.1110
Experience	Windows Feature Experience Pack 120.2212.3530.0

**Comment by dae** [comment]
I'm not able to reproduce this - both the French and French(Switzerland) voices listed in the Windows 10 preferences work on my machine. What does {{tts-voices:}} show? And do get you such an error message in 2.1.45?

**Comment by fuquasteve** [comment]
{{tts-voices:}}
produces no output on 2.1.45.

Reviewing a card with 
{{tts fr_FR:L2}}

Produces

Error 
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed. 
If the issue only occurs when add-ons are enabled, please use the Tools>Add-ons menu item to disable some add-ons and restart Anki, repeat until you discover the add-on that is causing the problem. 
When you've discovered the add-on that is causing the problem, please report the issue on the add-ons section of our support site. 
Debug info:
Anki 2.1.45 (355e66e8) Python 3.8.6 Qt 5.14.2 PyQt 5.14.2
Platform: Windows 10
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2021-07-29 19:58:59

Caught exception:
Traceback (most recent call last):
  File "aqt\webview.py", line 41, in cmd
  File "aqt\webview.py", line 139, in _onCmd
  File "aqt\webview.py", line 592, in _onBridgeCmd
  File "aqt\overview.py", line 91, in _linkHandler
  File "aqt\main.py", line 672, in moveToState
  File "aqt\main.py", line 694, in _reviewState
  File "aqt\reviewer.py", line 140, in show
  File "aqt\reviewer.py", line 160, in refresh_if_needed
  File "aqt\reviewer.py", line 209, in nextCard
  File "aqt\reviewer.py", line 324, in _showQuestion
  File "aqt\sound.py", line 153, in play_tags
  File "aqt\sound.py", line 204, in _play_next_if_idle
  File "aqt\sound.py", line 207, in _play
  File "aqt\sound.py", line 218, in _best_player_for_tag
  File "aqt\tts.py", line 110, in rank_for_tag
  File "aqt\tts.py", line 75, in voice_for_tag
  File "aqt\tts.py", line 71, in voices
  File "aqt\tts.py", line 489, in get_available_voices
  File "aqt\tts.py", line 494, in _voice_to_object
  File "<COMObject <unknown>>", line 3, in GetAttribute
pywintypes.com_error: (-2147352567, 'Exception occurred.', (0, None, None, None, 0, -2147200966), None)

I'm going to try to figure out exactly what is going on with my Windows set up.

**Comment by fuquasteve** [comment]
Got it!
Anki seemed to be keeping an old version of espeak (1.48) open on my windows machine.  I was unistalling tts programs from the computer in case they were confusing anything.  Espeak would not delete because it was open.  When I closed anki, espeak uninstalled, and when I reopened anki.  TTS worked as designed.
Thanks for the support.  I believe than anki may get confused by espeak sometimes.
